utils/linkage.py

```diff
@@ -2,108 +2,49 @@
 Module for performing record linkage on state campaign finance dataset
 """
 
-
-def calculate_string_similarity(string1: str, string2: str) -> float:
-    """Returns how similar two strings are on a scale of 0 to 1
-
-    The exact meaning of the metric is open, but the following must hold true:
-    1. equivalent strings must return 1
-    2. strings with no similar characters must return 0
-    3. strings with higher intuitive similarity must return higher scores
-
-    Args:
-        string1: any string
-        string2: any string
-    Returns:
-        similarity score
-
-    Sample Usage:
-    >>> calculate_string_similarity("exact match", "exact match")
-    1.0
-    >>> calculate_string_similarity("aaaaaa", "bbbbbbbbbbb")
-    0.0
-    >>> similar_score = calculate_string_similarity("very similar", "vary similar")
-    >>> different_score = calculate_string_similarity("very similar", "very not close")
-    >>> similar_socre > different_score
-    True
-    """
-    pass
-
-
-def get_likely_name(first_name: str, last_name: str, full_name: str) -> str:
-    """Given name related columns, return a person's likely name
-
-    Given different formatting used accross states, errors in data entry
-    and missing data, it can be difficult to determine someone's actual
-    name. For example, some states have a last name column with values like
-    "Doe, Jane", where the person's first name appears to have been erroneously
-    included.
-
-    Args:
-        first_name: raw value of first name column
-        last_name: raw value last name column
-        full_name: raw value of name or full_name column
-    Returns:
-        The most likely full name of the person listed
-
-    Sample Usage:
-    >>> get_likely_name("Jane", "Doe", "")
-    "Jane Doe"
-    >>> get_likely_name("", "", "Jane Doe")
-    "Jane Doe"
-    >>> get_likely_name("", "Doe, Jane", "")
-    "Jane Doe"
-    >>> get_likely_name("Jane Doe", "Doe", "Jane Doe")
-    "Jane Doe"
-    """
-    pass
+import usaddress
+
+
+def address_components(address_tuples):
```

Comment:
let's add documentation here to match the format of the other functions (add data type notation to the input and output, docstring, etc) and some sample tests.